code/game/objects/items/storage/firstaid.dm

```diff
@@ -521,11 +521,15 @@
 	if(issynth(target))
 		return FALSE
 
-	//ai_brain.appraise_inventory() Zonenote: Maybe needed idk
-
 	if(!length(contents) || !COOLDOWN_FINISHED(ai_brain, pill_use_cooldown))
 		return FALSE
 
+	var/obj/item/reagent_container/pill/pill = contents[1]
+	var/datum/reagent/reagent_datum = GLOB.chemical_reagents_list[pill.pill_initial_reagents[1]]
+
+	if((target.reagents.get_reagent_amount(reagent_datum.id) + 15) > REAGENTS_OVERDOSE)
```

Comment:
Couldn't we check how much is in the pill instead of a hardcoded 15

Reply:
lemme do it real fast then